mp/src/game/shared/neo/neo_player_shared.h

```diff
@@ -16,13 +16,7 @@
 
 #include "neo_predicted_viewmodel.h"
 
-#ifdef INCLUDE_WEP_PBK
```

Comment:
Seems unnecessary to require 64 bits here, since we only use 32 (without the PBK).

Reply:
so previously were we using 32 with the PBK? Why the long long int in the first place?

Reply:
Because previously we were using C++11, and so we needed 1 extra bit to avoid the UB of flipping the sign bit.

Reply:
0x0 is still invalid yes?

Reply:
> 0x0 is still invalid yes?

Yes, although we might not need to have it if it's not being referred to anywhere except for that old assertion.